client/components/statsTrack.jsx

```diff
@@ -1,18 +1,35 @@
 import React from 'react';
-import Moment from 'moment';
+import moment from 'moment';
 import BackButton from './BackButton';
 import ProgressBar from './ProgressBar';
 import style from '../style.css';
 
-class StatsTrack extends React.Component {
+const HOST = 'http://localhost:3002';
```

Comment:
This is quite the heavy refactor of the front-end, but I think you managed to make this file more legible and slightly more modular while maintaining the same functionality. Good job!